src/Lorentz/Contracts/BaseDAO/Types.hs

```diff
@@ -103,7 +103,7 @@ data Config proposalMetadata = Config
 defaultConfig :: Config pm
 defaultConfig = Config
   { cDaoName = "BaseDAO"
-  , cDaoDescription = [md|"BaseDAO description"|]
+  , cDaoDescription = [md|BaseDAO description.|]
```

Comment:
I think it should be something more descriptive, like, I don't know, at least "An example of a very simple DAO contract without any custom checks, extra data and decision lambda".